content/opt/run

```diff
@@ -219,5 +212,5 @@ echo "==================================================================="
 touch ${initfile}
 fi
 
-echo "Starting Supervisor.  You can safely CTRL-C and the container will continue to run with or without the -d (daemon) option"
+echo "Starting Supervisor."
```

Comment:
I experienced differently. Every time i hit Ctrl+C it stops.

Probably this is, because docker run bridges the sequence to the supervisor.

using docker 1.12